x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/policy_template_form.tsx

```diff
@@ -83,7 +85,13 @@ export const CspPolicyTemplateForm = memo<Props>(({ newPolicy, onChange, edit })
 
   return (
     <div>
+      <IntegrationSettingsInfo type={input.policy_template} showStepTitle={!!edit} />
       <PolicyInputSelector input={input} setInput={setEnabledPolicyInput} disabled={!!edit} />
+      <IntegrationSettings
+        name={newPolicy.name}
+        description={newPolicy.description || ''}
+        onChange={(field, value) => updatePolicy({ ...newPolicy, [field]: value })}
+      />
```

Comment:
A note regarding the design, IDK if it was agreed with product/design, but i noticed it was different from the figmas. 
Personally, I think the figma design is easier on the eyes, even though not perfectly logical in terms of UI building

More specifically, 
![image](https://user-images.githubusercontent.com/51442161/214811577-49ae1eee-9650-4cba-a99c-c4dcfc795d8e.png)

In the design, there is this pattern of 
1. Title
2. Description
3. Select boxes
4. fields

which repeats itself for both steps

in the implementation

![image](https://user-images.githubusercontent.com/51442161/214812537-2cb9cec6-30a2-43fd-b02e-856d889eded9.png)

This pattern is broken, its different for each step.
Also, the titles are not the same size and the space between the title and description is too large
(not counting pixels, it just looks like we need to use a smaller spacer size)

Reply:
good catch! 💯 

1 - the `setup access` moved below the `name` / `description` fields and its size changed to match the first title. 
2 - the space between the `Configure Integration` and our first component with the text description for the inputs, which is a big big, but is set by the eui steps component, which we render inside of it, so no control on that. 

will take a look at all usages of `EuiSpacer` to see there isn't a rouge one

![Screen Shot 2023-01-26 at 14 26 47](https://user-images.githubusercontent.com/20814186/214835830-35ff043c-0c31-416f-b910-28e00faaec0e.png)